src/Api/ResetPasswordApiController.php

```diff
@@ -34,38 +34,38 @@ class ResetPasswordApiController extends BaseApiController
     /**
      * Get the response for a successful password reset.
      *
-     * @param  string  $response ID of language resource to send as response
+     * @param string  $response ID of language resource to send as response
```

Comment:
Unnecessary two spaces. 
Also, INHO, "*ID of language resource to **use** as response*" (just suggestion)